src/Microsoft.Health.Dicom.Blob/Features/ExternalStore/ExternalBlobClient.cs

```diff
@@ -34,9 +36,42 @@ internal class ExternalBlobClient : IBlobClient
         _credentialProvider = EnsureArg.IsNotNull(credentialProvider, nameof(credentialProvider));
         _blobClientOptions = EnsureArg.IsNotNull(blobClientOptions?.Value, nameof(blobClientOptions));
         _externalStoreOptions = EnsureArg.IsNotNull(externalStoreOptions?.Value, nameof(externalStoreOptions));
+        _externalStoreOptions.ServiceStorePath = SanitizeServiceStorePath(_externalStoreOptions.ServiceStorePath);
```

Comment:
Can we put more comments on how this is passed through? Right now the RP configuration is just containerName no path.

Also validation done here is too late, it needs to be done in RP code.